src/System.Data.Common/src/System/Data/CommandBehavior.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+
+//------------------------------------------------------------------------------
+
+namespace System.Data
+{
+    [Flags()]
+    public enum CommandBehavior
+    {
+        Default = 0,  // with data, multiple results, may affect database, MDAC 68240
+        SingleResult = 1,  // with data, force single result, may affect database
+        SchemaOnly = 2,  // column info, no data, no effect on database
+        KeyInfo = 4,  // column info + primary key information (if available)
+        // 
```

Comment:
These comments have been cleaned up. 
They came out as a by product of the sanitizer.